src/components/Avatar/Avatar.tsx

```diff
@@ -6,17 +6,34 @@ import {
 } from "@radix-ui/react-avatar";
 import styled from "styled-components";
 
+type Size = "md" | "sm";
 type TextSize = "md" | "sm";
+type State = "neutral" | "success" | "vivid";
```

Comment:
Where are we using vivid state?

Reply:
The notification bell, for example, when it shows unread notifications.